roles/reproducer/tasks/configure_crc.yml

```diff
@@ -5,6 +5,60 @@
   ansible.builtin.slurp:
     path: /etc/ci/env/networking-environment-definition.yml
 
+- name: Configure CRC services
+  delegate_to: crc-0
+  vars:
+    _parsed: "{{ _net_conf.content | b64decode | from_yaml}}"
+    _crc: "{{ _parsed.instances['crc-0'].networks.ctlplane }}"
+  block:
+    - name: Ensure crc-0 knows about its second NIC
+      become: true
+      vars:
+        _ctlplane: "{{ _parsed.networks.ctlplane }}"
+      community.general.nmcli:
+        autoconnect: true
+        conn_name: private_net
+        ifname: "{{ _crc.interface_name }}"
+        type: ethernet
+        ip4: "{{ _crc.ip_v4 }}/24"
+        never_default4: true
+        state: present
+
+    - name: Check which dnsmasq config we must edit
+      register: _dnsmasq
+      ansible.builtin.stat:
+        path: '/srv/dnsmasq.conf'
+        get_attributes: false
+        get_checksum: false
+        get_mime: false
+
+    - name: Configure local DNS for CRC pod
```

Comment:
iirc that's more or less covered by @lewisdenny PR (though I just saw he closed #1944)?